incubator/elasticsearch/values.yaml

```diff
@@ -3,45 +3,52 @@
 # Declare variables to be passed into your templates.
 
 image:
-  repository: "jetstack/elasticsearch-pet"
-  tag: "2.4.0"
+  repository: "centerforopenscience/elasticsearch"
+  tag: "5.4"
+  # tag: "2.4"
   pullPolicy: "Always"
 
 cluster:
   name: "elasticsearch"
   config:
-    
+  env:
+    # IMPORTANT: https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html#minimum_master_nodes
+    # To prevent data loss, it is vital to configure the discovery.zen.minimum_master_nodes setting so that each master-eligible
+    # node knows the minimum number of master-eligible nodes that must be visible in order to form a cluster.
```

Comment:
Maybe we can add a FAQ like https://github.com/pires/kubernetes-elasticsearch-cluster#why-does-number_of_masters-differ-from-number-of-master-replicas